src/test/resources/processor_test_files/S4065_ThreadLocalWithInitial/ThreadLocalInitial.java

```diff
@@ -0,0 +1,13 @@
+
+public class ThreadLocalInitial {
```

Comment:
We can rename this, I think.

```suggestion
public class ThreadLocalInitialWithReturnStatement {
```